src/app/wizard/step/provider-settings/provider/extended/openstack/default/component.ts

```diff
@@ -181,8 +224,13 @@ export class OpenstackProviderExtendedDefaultCredentialsComponent
     this._unsubscribe.complete();
   }
 
-  subnetIDDisplayName(id: string): string {
-    const subnetID = this.subnetIDs.find(subnetID => subnetID.id === id);
+  ipv4SubnetIDDisplayName(id: string): string {
```

Comment:
can we move to shared dir `src/app/wizard/step/provider-settings/provider/extended/openstack/shared`?
to avoid duplication? or I guess we will still need methods to to set values within a component.

thoughts?

- ipv4SubnetIDDisplayName
- ipv6SubnetIDDisplayName
- _loadSecurityGroups
- _loadNetworks
- _loadSubnetIDs

Reply:
There's lot more code duplication between this component and `applications/component.ts` so we have to refactor that in a separate ticket. Regarding these methods, this structure has been followed in other provider components too so we'll have to refactor that in separate ticket if we want.